packages/amplify-migration-tests/src/__tests__/migration_tests_v10/__snapshots__/api-graphql-v1.migration.test.ts.snap

```diff
@@ -0,0 +1,10 @@
+// Jest Snapshot v1, https://goo.gl/fbAQLP
+
+exports[`api graphql v1 migration tests ...adds graphql with v1 transformer, updates models, adds function access, and pulls with latest version 1`] = `""`;
+
+exports[`api graphql v1 migration tests ...adds graphql with v1 transformer, updates models, adds function access, and pulls with latest version 2`] = `
+"Outputs
+[+] Output LambdaExecutionRoleArn: {\\"Value\\":{\\"Fn::GetAtt\\":[\\"LambdaExecutionRole\\",\\"Arn\\"]}}
```

Comment:
This LambdaExecutionRoleArn is new in 10.5.2 and forward, so this is good (we are expecting it to create this in vLatest)

The migration tests are setup with 10.5.1, which does not have this.